### PR TITLE
Fixes some adminhelp issues

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Removes the ability for players to close their own tickets and open a new one. This is unnecessary, and very annoying for the admins.
Improves handling of disconnected clients in the middle of an ahelp reply. If the (non-admin) recipient has disconnected or relogged, will attempt to send to new client or fallback to printing the message in chat and in the ticket for easy copying later.
Sending a PM to another admin didnt create a ticket, since admins do get legitimately bwoinked some times its now changed so it always does. This also cleans up a runtime with admin -> admin PM's.

## Why It's Good For The Game
Gets rid of and fixes some annoyances.

## Changelog
:cl: Skoglol
admin: Adminhelps has gotten a little bit smarter. Improved handling of target dis- and re-connections, admin->admin PM now opens a ticket, players can no longer close their own tickets.
/:cl:

### CODE REVIEW

**Comment by ShizCalev** [comment]
>Removes the ability for players to close their own tickets and open a new one. This is unnecessary, and very annoying for the admins.

is this equivalent to saying "i want to speak to your manager"

**Comment by kriskog** [comment]
Pretty much, yeah :joy:

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -510,16 +510,12 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
...
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Adminhelp") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 	if(current_ticket)
-		if(alert(usr, "You already have a ticket open. Is this for the same issue?",,"Yes","No") != "No")
-			if(current_ticket)
-				current_ticket.MessageNoRecipient(msg)
-				current_ticket.TimeoutVerb()
-				return
-			else
-				to_chat(usr, "<span class='warning'>Ticket not found, creating new one...</span>", confidential = TRUE)
+		if(current_ticket)
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminhelp.dm:513]: 🤔

Comment (kriskog) [reply code/modules/admin/verbs/adminhelp.dm:513]: For when you want to be really REALLY sure

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**